build/metadata/github-actions/README.md

```diff
@@ -34,6 +38,34 @@ Commit this file under a new branch.  Submit as a PR to your code reviewers to b
 
 As shown in the above example, the workflow should run on the `push` event, and contain an action provided by the [launchdarkly/find-code-references repository](https://github.com/launchdarkly/find-code-references). The `LD_ACCESS_TOKEN` configured in the previous step should be included as a secret, as well as a new environment variable containing your LaunchDarkly project key.
 
+## Additional Examples
+The example below is the same as first, but it also excludes any `dependabot` branches. We suggest excluding any automatically generated branches where flags do not change.
+
+```yaml
+on:
+  push:
+    branches-ignore:
+      - 'dependabot/**'
+
+name: Find LaunchDarkly flag code references
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  launchDarklyCodeReferences:
+    name: LaunchDarkly Code References
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 10 # This value must be set if the lookback configuration option is not disabled for find-code-references. Read more: https://github.com/launchdarkly/ld-find-code-refs#searching-for-unused-flags-extinctions
+    - name: LaunchDarkly Code References
+      uses: launchdarkly/find-code-references@vx.x.x
+      with:
+        accessToken: ${{ secrets.LD_ACCESS_TOKEN }}
+        projKey: YOUR_PROJECT_KEY
```

Comment:
thoughts on changing this in both examples?

```suggestion
        projKey: LD_PROJECT_KEY
```

Reply:
Changing in both works for me.